### PR TITLE
docs(react-instantsearch): remove mentions of ris

### PR DESCRIPTION
**Pull request by vvo**
Since we moved react-instantsearch to
https://github.com/algolia/react-instantsearch/ and
https://community.algolia.com/react-instantsearch/

We can now remove mentions of the old repo organisation

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,16 +1,44 @@
 <!--
+  ***************************
   Thanks for participating in this project!
-  Please:
-    - make sure you are using the latest version of the library.
-    - do at least one GitHub search in current issues, maybe your question is already here
+
+  This form is to report issues or new features.
+  As for general questions like "How to do routing using React InstantSearch",
```
Comment (Haroenv) [inline .github/ISSUE_TEMPLATE.md]: This still has `React`

File: README.md [hunk 2]
```diff
@@ -1,209 +1,26 @@
...
+[![InstantSearch.js logo][readme-logo]][website]
 
-[![Version][version-svg]][package-url] [![Build Status][travis-svg]][travis-url] [![License][license-image]][license-url] [![Downloads][downloads-image]][downloads-url]
+InstantSearch projects: **InstantSearch.js** [React InstantSearch][react-instantsearch-github], [InstantSearch Android](instantsearch-android-github).
 
-**instantsearch.js** is a library of UI widgets to help you build the best instant-search experience with [Algolia's Hosted Search API](https://www.algolia.com/?utm_medium=social-owned&utm_source=GitHub&utm_campaign=InstantSearch%20repository).
+## InstantSearch.js
 
-Have a look at the website: [https://community.algolia.com/instantsearch.js/][instantsearch-website].
+> ⚡ Lightning-fast search for websites
```
Comment (Haroenv) [inline README.md]: maybe a small link to `react-instantsearch` or to the upcoming landing page for instantsearch here?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (vvo) [inline .github/ISSUE_TEMPLATE.md]: fixed

File: README.md (hunk 2 above)
Comment (vvo) [inline README.md]: There's one here: https://github.com/algolia/instantsearch.js/blob/c3fad4c6082fa7586bad2554e693db797aca420a/README.md

File: README.md [hunk 3]
```diff
@@ -1,209 +1,26 @@
...
-Your docker installation must be compatible and ready to make the [--net="host"](https://docs.docker.com/engine/reference/run/#network-host) works.
-
-## License
-
-instantsearch.js is [MIT licensed](./LICENSE).
-
-## Contributing
-
-We have a [contributing guide](CONTRIBUTING.md), join us!
+[logo]: ./docgen/readme-logo.png
```
Comment (Haroenv) [inline README.md]: This file doesn’t exist

**Comment by Haroenv** [comment]
Travis fails because this PR is done to `master`, and not to `develop`

```
./scripts/validate-pr-done-on-develop.sh
Error: Pull request cannot be done on master
```

**Comment by vvo** [comment]
Thanks for all the review @Haroenv, much appreciated because I forgot many things :)

**Pull request merged**